log/example_new_json_logger_test.go

```diff
@@ -0,0 +1,20 @@
+package log_test
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/go-kit/kit/log"
+)
+
+// JSONLogger Example
+func ExampleNewJSONLogger() {
+	var buf bytes.Buffer
+
+	logger := log.NewJSONLogger(&buf)
+	logger.Log("question", "what is the meaning of life?", "answer", 42)
+
+	fmt.Print(&buf)
+	// Output:
+	// {"answer":42,"question":"what is the meaning of life?"}
+}
```

Comment:
Good point. I couldn't find it documented anywhere. In fact, the spec specifically says the keys are unordered. From https://www.ietf.org/rfc/rfc4627.txt:
>An object is an unordered collection of zero or more name/value pairs, where a name is a string and a value is a string, number, boolean, null, object, or array.